techniques/system/common/1.0/promises.st

```diff
@@ -190,7 +190,7 @@ bundle agent check_disable_agent
 
       # Only define this class when we're ready to die - this is a special class name in "abortclasses"
       "should_not_continue"
-        expression => "should_disable_agent",
+        expression => "should_disable_agent.!ignore_disable_agent",
```

Comment:
OK, just saw your comment in #996, that makes for a reasonable case. I hadn't understood "disable inventory" is supposed to *prevent* all and any inventories, I imagined it was just to stop automatic (nightly) inventories. Let me think this over.